### ISSUE TITLE
Incorrect BigInt processing in web console

### ISSUE THREAD

**Issue by VadimEv** [freeCodeCamp/freeCodeCamp#39352]
As example, in Euler-48 challenge incorrect type error is shown: there are no conversion of BigInt object to number - instead first conversed to string, sliced and only after transformed  to number.
![Euler-48-error](https://user-images.githubusercontent.com/10043733/89102398-70c96200-d411-11ea-987d-81ab64a532c2.png)
I do believe it's some kind of glitch with console, not BigInt object itself,  because one-liner for Euler-16 works just fine, as well as Euler-20:
![Euler-16-correct](https://user-images.githubusercontent.com/10043733/89102505-9efb7180-d412-11ea-94de-f46a104dc293.png)
however very same approach fails at Euler-56 again.
Codes for reference:
16: https://github.com/VadimEv/ProjectEuler100/blob/master/Euler-16.js
20: https://github.com/VadimEv/ProjectEuler100/blob/master/Euler-20.js
48: https://github.com/VadimEv/ProjectEuler100/blob/master/Euler-48-clean.js
56: https://github.com/VadimEv/ProjectEuler100/blob/master/Euler-56.js (result changes type, but if it initially a number - it doesn't help)
if run in clean browser console - everything works just fine.

**Comment by Bouncey** [comment]
I have confirmed the issue is in the the test runner. The provided solution working in the developer console.

**Comment by nhcarrigan** [comment]
Constructing the code by hand, the TypeError gets thrown on the `sum += i ** i` line. I explored other variations, and it appears the use of the `**` operator is causing the error. [According to the MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#Operators), the `BigInt` type *should* support the `**` operator - as @Bouncey mentioned this code does work in my browser console. 

I don't believe it's an error in the test runner, but in the Monaco editor itself?

**Comment by ojeytonwilliams** [comment]
I think it's a issue with how Babel transforms exponentials.  Take a look at this [repl](https://babeljs.io/repl#?browsers=&build=&builtIns=false&spec=true&loose=true&code_lz=DYUwLgBAzgrgthAvBAQgSwOYEkB2YAUATAJQBQscAVJRQNylA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Cenv&prettier=false&targets=&version=7.11.6&externalPlugins=)

I found a post that suggests we might be able to fix it via [plugins](https://stackoverflow.com/questions/57902010/how-to-use-bigint-exponentiation-with-babel), so I'll give that a shot.  Thanks for looking into this @Bouncey and @nhcarrigan!

**Comment by ojeytonwilliams** [comment]
Unfortunately we would need to start using the JSBI to handle BigInts if we want to support older browsers.  Instead, I think we should make the browsers we say we support (on /news/support) the ones we explicitly support (via browserlist).  I'll create a separate issue to discuss this, since it's a bit broader than BigInt.

**Comment by raisedadead** [comment]
Can we confirm the triage state on this or close if decayed?

**Comment by nhcarrigan** [comment]
I can confirm that the code Oliver provided in the linked repl above still throws a conversion error.

```js
let sum = BigInt(2)
sum**sum;
```

However, I think this could potentially be closed as a sub-issue of #39647

**Comment by PabloPicas0** [comment]
I know its closed issue but if anyone came here to find solution here is workaround

 `function power(base, exponent) {
  base = BigInt(base)
  let result = 1n;
  for (let count = 0; count < exponent; count++) {
    result *= base;
  }
  return result;
};`
